### PR TITLE
62/Add NetworkDefinition.infoUrl, statsUrl, add UI for urls

### PR DESCRIPTION
**Pull request by friedger**
Add NetworkDefinition.infoUrl, statsUrl, add UI for urls

towards #62: icons could be improved, urls for ropsten to be verified

screenshot_1511968020

### CODE REVIEW

File: app/src/main/java/org/walleth/data/networks/RopstenNetworkDefinition.kt [hunk 1]
```diff
@@ -804,4 +804,8 @@ class RopstenNetworkDefinition : BaseNetworkDefinition() {
     "coinbase": "0x0000000000000000000000000000000000000000"
 }
         """
+
+    override val statsUrl = "https://ropsten.etherscan.io/"
```
Comment (ligi) [inline app/src/main/java/org/walleth/data/networks/RopstenNetworkDefinition.kt]: I think this should be null as AFAIK ropsten has no stats server

File: app/src/main/java/org/walleth/ui/NetworkDefinitionViewHolder.kt [hunk 2]
```diff
@@ -7,14 +7,22 @@ import org.walleth.data.networks.NetworkDefinition
...
         itemView.setOnClickListener {
             onClickAction.invoke(networkDefinition)
         }
 
+        itemView.info_indicator.setOnClickListener {
+            onInfoClickAction.invoke(networkDefinition, false)
+        }
+
+        itemView.stats_indicator.setOnClickListener {
+            onInfoClickAction.invoke(networkDefinition, true)
```
Comment (ligi) [inline app/src/main/java/org/walleth/ui/NetworkDefinitionViewHolder.kt]: I think we should have 2 lambdas here for readability - onInfoClick and onStatsClick

File: app/src/main/java/org/walleth/data/networks/RopstenNetworkDefinition.kt [hunk 3]
```diff
@@ -804,4 +804,8 @@ class RopstenNetworkDefinition : BaseNetworkDefinition() {
...
+
+    override val infoUrl = "https://ropsten.etherscan.io/"
```
Comment (ligi) [inline app/src/main/java/org/walleth/data/networks/RopstenNetworkDefinition.kt]: not yet sure about this url - thinking about: https://github.com/ethereum/ropsten

**Comment by friedger** [comment]
Requested changes addressed in https://github.com/walleth/walleth/pull/125/commits/dd677dba270ff16b023f05415f97a254e36d88fb

File: app/src/main/java/org/walleth/data/networks/RinkebyNetworkDefinition.kt [hunk 4]
```diff
@@ -17,4 +17,8 @@ class RinkebyNetworkDefinition : BaseNetworkDefinition() {
     override val bootNodes = listOf("enode://a24ac7c5484ef4ed0c5eb2d36620ba4e4aa13b8c84684e1b4aab0cebea2ae45cb4d375b77eab56516d34bfbd3c1a833fc51296ff084b770b94fb9028c4d25ccf@52.169.42.101:30303?discport=30304")
 
     override val genesis = ""
+
+    override val statsUrl = "https://stats.rinkeby.io/"
+
+    override val infoUrl = "https://www.rinkeby.io/"
```
Comment (ligi) [inline app/src/main/java/org/walleth/data/networks/RinkebyNetworkDefinition.kt]: in the end both urls point to the same thing - so I think I will rework this a bit after the merge - I think one link might be enough context

**Comment by ligi** [comment]
:+1:

**Pull request merged**